sei-db/ss/pebbledb/db.go

```diff
@@ -338,7 +345,18 @@ func (db *Database) Has(storeKey string, version int64, key []byte) (bool, error
 	return val != nil, nil
 }
 
-func (db *Database) Get(storeKey string, targetVersion int64, key []byte) ([]byte, error) {
+func (db *Database) Get(storeKey string, targetVersion int64, key []byte) (_ []byte, _err error) {
+	startTime := time.Now()
+	defer func() {
+		otelMetrics.getLatency.Record(
+			context.Background(),
+			time.Since(startTime).Seconds(),
+			metric.WithAttributes(
+				attribute.Bool("success", _err == nil),
+				attribute.String("store", storeKey),
```

Comment:
Double checking: What does the cardinality of `storeKey` look like? How many unique store keys for the lifetime of the program?